File: uva11854/Kotlin/uva11854.kt
/*
    @author: Nicolas Cardozo
    @complexity: O(1) for each test case
    @time:
    @memory: 
 */

import java.io.BufferedReader
import java.io.InputStreamReader

fun main(args : Array<String>) {
    val In = BufferedReader(InputStreamReader(System.`in`))
    while(true) {
        val lengths = In.readLine().split(" ").map{x -> x.toInt()}.toMutableList()
        if(lengths[0] == 0 && lengths[1] == 0 && lengths[2]==0) {
            break
        } else {
            lengths.sort()
            if(lengths[0]*lengths[0] + lengths[1]*lengths[1] == lengths[2]*lengths[2]) {
                println("right")
            } else {
                println("wrong")
            }
        }
    }
}

File: uva/11854/uva11854.py
# @author: Nicolas Cardozo
# @complexity: O(1) for each test case
# @running_time: 0.000
# @memory: 

from sys import stdin, stdout

def main():
    while(True):
        sides = stdin.readline().split(" ")
        ints = [int(x) for x in sides]
        if(ints[0]==0 and ints[1]==0 and ints[2]==0):
            break
        else:
            ints.sort()
            if(ints[0]*ints[0] + ints[1]*ints[1] == ints[2]*ints[2]):
                stdout.write("right\n")
            else:
                stdout.write("wrong\n")

main()
 